frontend/src/pages/AppointmentStepperPage.tsx

```diff
@@ -0,0 +1,9 @@
+import { AppointmentScheduler } from '../components/appointment-scheduler/AppointmentScheduler';
+
+export default function AppointmentStepperPage() {
+  return (
+    <section className="flex flex-col gap-2 p-3 md:p-10">
```

Comment:
Cuando se usa flex-col por defecto no viene centrado
```suggestion
    <section className="flex flex-col items-center gap-2 p-3 md:p-10">
```